src/scss/mixins/layout/_grid.scss

```diff
@@ -0,0 +1,110 @@
+// Utilities
+@import "column";
+
+// Grid mixin to be applied on container level - controls size of children
+@mixin grid($layout: full, $gutter: $base-gutter*0.5) {
```

Comment:
I'm considering this syntax.
```
@include grid(1 2 1 3, 20px);
```
So your `one-third` would be `(1 3)` ;

What do you think about that @christofferberg ?

Reply:
So one-third-two-thirds would be ```@include grid(1 3 2 3, 20px);```

I don't like it - I think we can do better.

Reply:
I see the grid as a sum of all the items divided by it's number.
So `one-third-two-thirds` would be `1 2`.  

**Like this**
```
1+2=3;
1/3 = 0.33;
2/3 = 0.66;
```

Does that make sense? :)